app/containers/Footer/index.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Most probably it is going to be static component. Should be component, not container.